db_init.py

```diff
@@ -42,14 +82,28 @@
         if db.inspect(db.engine).get_table_names():
             print("Tables already exist.")
             # clear the codes table
-            db.session.query(Codes).delete()
+            db.session.delete(Codes)
             db.session.commit()
             sys.exit()
         db.create_all()
 
 elif sys.argv[1] == "clear":
     with app.app_context():
         db.drop_all()
+        
+elif sys.argv[1] == "addCourses":
+    with app.app_context():
+        db.create_all()
+
+        json_url = "https://raw.githubusercontent.com/quacs/quacs-data/master/semester_data/202409/catalog.json"
```

Comment:
can you update this function to intake the URL as an argument, should check if `sys.argv[2]` is present (add logic to confirm extra arguments exist). Add logic to confirm that it's a URL.